.github/workflows/ci.yml

```diff
@@ -13,6 +13,7 @@ env:
   RUST_BACKTRACE: 1
   CARGO_TERM_COLOR: always
   CLICOLOR: 1
+  TERM: xterm
```

Comment:
This still needed?

Reply:
Yeah, still need it for bash. Otherwise, in exhaustive it would output:
```
% cmd 
-- help
...
% cmd 
```

Reply:
Should we set it in completest?  Our pty allows some level of "smarts"

Reply:
I guess we could set it is the default TERM is None. 
[TERM][1] [environment variable][2] is to specify terminal emulator's capabilities.
We run these complete tests on Linux CI only so we could control/guess the terminal type.
Maybe people shouldn't run `compiletest` on non-modern terminal?
What do you think?

[1]: https://man7.org/linux/man-pages/man7/term.7.html
[2]: https://bash.cyberciti.biz/guide/$TERM_variable#Purpose

Reply:
Keep in mind that we aren't exposing the terminal to bash but a psuedo-pty we control.  The user's terminal capabilities do not matter.

Reply:
But isn't complete tests is outputted/run on the host terminal.
The "psuedo-pty we control" is just a thin wrapper around the host terminal. 
Anyway, I opened https://github.com/assert-rs/completest/pull/40 .